### ISSUE TITLE
How can I get "single spaced" translation?

### ISSUE THREAD

**Issue by captainload** [speech-translator-ext/speech-translator-readme#14]
Between every line of subtitle there is a blank line.  I've tried resizing the window and changing font size but neither has any effect.  Is there a way to switch to single-spaced output?

**Comment by kappaflow** [comment]
Yes, you can make the view more compact in case of disabling the transcript displaying. The default setup shows transcript right below the translation, so the space is reserved.
